src/index.html

```diff
@@ -11,7 +11,190 @@
+        <legend>Registration</legend>
+
+        <label for="your-mail" class="inputs top">
+          E-mail:
+
+          <input type="email"
+            placeholder="email@example.com"
+            id="your-mail"
+            multiple
+          >
+        </label>
+
+        <label for="your-password" class="inputs bottom">
+          Password
+
+          <input type="password"
+            required
+            minlength="6"
+            maxlength="35"
+            id="your-password"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="blocks">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="inputs top">
+          Do you love cats?
+
+          <label for="loving-cats">
+            <input type="radio"
+              name="cats"
+              id="loving-cats"
+              value="yes"
+            >
+            Yes
+          </label>
+
+          <label for="not-loving-cats">
+            <input type="radio"
+              name="cats"
+              id="not-loving-cats"
+              value="no"
+            >
+            No
+          </label>
+        </div>
+
+        <label for="your-color" class="inputs">
+          What is your favorite color?
+
+          <input type="color"
+            name="color"
+            id="your-color">
+        </label>
+
+        <label for="going-sleep" class="inputs">
+          What time do you go to bed?
+
+          <input type="time"
+            name="time"
+            id="going-sleep">
+        </label>
+
+
+        <label for="your-car" class="inputs">
+          What are your favorite brand of cars?
+
+          <!-- <input type="text"
+            name="cars"
+            id="your-car"
+            value="BMW Audi Lada"
+          > -->
+          <select name="cars"
+            id="your-car"
+            multiple
+          >
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+            <option value=""></option>
+          </select>
+        </label>
+
+        <label for="rating" class="inputs bottom">
+          How do you rate our work?
+
+          <input type="range"
+            name="rate"
+            id="rating"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="blocks">
+        <legend>Additional info:</legend>
+
+        <label for="your-comments" class="inputs top">
+            Comments:
+
+          <textarea name="comments"
+            id="your-comments"
+            rows="2"
+            cols="20"
+         ></textarea>
+        </label>
+
+        <label for="your-recommends" class="inputs bottom">
+          Would you recommend us?
+
+          <select name="recommendations" id="your-recommends">
+            <option value="yes">yes</option>
+            <option value="no">no</option>
+          </select>
+        </label>
+      </fieldset>
+
+
+      <button type="submit">Submit</button>
```

Comment:
```suggestion
      </fieldset>

      <button type="submit">Submit</button>
```
one blank line maximum